packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -112,7 +112,7 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
       width: '100%',
       minHeight: DROPDOWN_ITEM_HEIGHT,
       lineHeight: 20,
-      height: 'auto',
+      height: isIE11 ? 0 : 'auto', // auto does not work properly in IE11
```

Comment:
Hmm. isIE11 is a function. This would always result with true, I believe. Am I missing something?

Reply:
Ah gotcha, my mistake. Yes, `height: 0` seems to work in all scenarios that I've tested. I'll change it to that!